### PR TITLE
feat: delete deployments by tag

### LINKED ISSUE #200: Multiple deployments: extend proxy config table to include branch/PR data
If we want to be able to delete all the deployments for a branch/PR after it has been deleted/closed, we need to store the information which deployment belongs to which branch/PR. This also requires for the deletion to be done inside a lambda based on uploading a file to a bucket, similar to how deployments are created.

It would probably also make sense to add `createdAt` and `notes` to the deployment table, so that users can add more metadata to the table, that's then also displayed in `list-deployments`. Users could for example want to delete the oldest deployments first.

### PR DESCRIPTION
**Pull request by endor**
Closes #200

### CODE REVIEW

**Review by ofhouse — COMMENTED** [review]
Just one thing about `console.table`, otherwise looks good!

File: packages/tf-next/package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "@vercel/routing-utils": "^1.10.1",
     "archiver": "^5.3.0",
     "aws-sdk": "*",
+    "console.table": "^0.10.0",
```
[the remark this answers is not in the document]
Comment (endor) [reply packages/tf-next/package.json]: Okay, changed that now. The output sadly also includes an `(index)`, which we don't really need, but it still looks nice:
Screenshot 2021-09-22 at 8 08 20

**Review by b6pzeusbc54tvhw5jgpyw8pwz2x6gs — COMMENTED** [review]
Since this is a command to delete one specific deployment, isn't the file name `delete-deployment.ts` rather than `delete-deployments.ts`?

**Comment by endor** [comment]
It can be used to both delete a single deployment via passing `deploymentId` or (possibly) multiple deployments when passing `tag`.

**Comment by b6pzeusbc54tvhw5jgpyw8pwz2x6gs** [comment]
If a tag can be attached to more than one distribution, you are right.

**Review by ofhouse — APPROVED** [review]

**Pull request merged**